tests/unit/packages/Microsoft.Bot.Components.Telephony.Tests/TimeoutChoiceInputTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Components.Telephony.Actions;
+using Microsoft.Bot.Schema;
+using Moq;
+using System.Threading.Tasks;
+using Xunit;
+using Microsoft.Bot.Builder.Dialogs.Memory;
+using Microsoft.Bot.Builder.Dialogs.Memory.Scopes;
+using System.Collections.Generic;
+using System.Threading;
+
+namespace Microsoft.Bot.Components.Telephony.Tests
+{
+    public class TimeoutChoiceInputTests
+    {
+        [Fact]
+        public async Task TimeoutChoiceInput_TurnCountIsCorrectlyUpdated()
+        {
+            // Setup
+            Mock<ITurnContext> mockTurnContext = new Mock<ITurnContext>();
```

Comment:
Could you please you use `var` for these `new`-ed up instances to be more consistent and brief in the code base? 😃 

I see additional explicit typing for `DialogStateManagerConfiguration`, `DialogContext`, and `TimeoutChoiceInputTestHelper`; where `var` could be used instead.